lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -7089,7 +7090,7 @@ namespace Js
     {
 #if ENABLE_PROFILE_INFO
         // Switch off profiling is asmJsFunction
-        if (this->GetIsAsmJsFunction())
+        if (this->GetIsAsmJsFunction() || this->GetAsmJsModuleInfo())
```

Comment:
can we use GetIsAsmjsMode() instead? otherwise it need a lock while accessing GetAsmJsModuleInfo() from background thread